src/com/xilinx/rapidwright/placer/dreamplacefpga/DREAMPlaceFPGA.java

```diff
@@ -0,0 +1,284 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.xilinx.rapidwright.placer.dreamplacefpga;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import com.xilinx.rapidwright.design.Design;
+import com.xilinx.rapidwright.device.Device;
+import com.xilinx.rapidwright.edif.EDIFNetlist;
+import com.xilinx.rapidwright.edif.EDIFTools;
+import com.xilinx.rapidwright.interchange.DeviceResourcesWriter;
+import com.xilinx.rapidwright.interchange.Interchange;
+import com.xilinx.rapidwright.interchange.LogNetlistWriter;
+import com.xilinx.rapidwright.interchange.PhysNetlistReader;
+import com.xilinx.rapidwright.interchange.PhysicalNetlistToDcp;
+import com.xilinx.rapidwright.tests.CodePerfTracker;
+import com.xilinx.rapidwright.util.FileTools;
+
+/**
+ * Utility wrapper class to facilitate using DREAMPlaceFPGA as an external
+ * placer.
+ */
+public class DREAMPlaceFPGA {
+
+    public static final String INTERCHANGE_NETLIST = "interchange_netlist";
+    public static final String INTERCHANGE_DEVICE = "interchange_device";
+    public static final String RESULT_DIR = "result_dir";
+    public static final String IO_PL = "io_pl";
+    public static final String GPU = "gpu";
+    public static final String NUM_BINS_X = "num_bins_x";
+    public static final String NUM_BINS_Y = "num_bins_y";
+    public static final String GLOBAL_PLACE_STAGES = "global_place_stages";
+    public static final String TARGET_DENSITY = "target_density";
+    public static final String DENSITY_WEIGHT = "density_weight";
+    public static final String RANDOM_SEED = "random_seed";
+    public static final String SCALE_FACTOR = "scale_factor";
+    public static final String GLOBAL_PLACE_FLAG = "global_place_flag";
+    public static final String ROUTABILITY_OPT_FLAG = "routability_opt_flag";
+    public static final String LEGALIZE_FLAG = "legalize_flag";
+    public static final String DETAILED_PLACE_FLAG = "detailed_place_flag";
+    public static final String DTYPE = "dtype";
+    public static final String PLOT_FLAG = "plot_flag";
+    public static final String NUM_THREADS = "num_threads";
+    public static final String DETERMINISTIC_FLAG = "deterministic_flag";
+    public static final String ENABLE_IF = "enable_if";
+    public static final String ENABLE_SITE_ROUTING = "enable_site_routing";
+
+    public static final String IO_PL_DEFAULT = "";
```

Comment:
```suggestion
    public static final String IO_PL_DEFAULT = ""; //EMPTY
```
I'm assuming we meant to have an empty string here.